website/markdown/docs/components/events-table.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import ReactMarkdown from 'react-markdown'
+
+const EventsTable = ({ args }) => {
```

Comment:
You are becoming a React expert @andreacipriani

Reply:
it's clearly a copy pasta 🍝